<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfTime</class>
 <widget class="QWidget" name="ConfTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ConfTime{
background-color: rgb(255, 255, 255);
}
QLineEdit{
font: 20pt &quot;文泉驿等宽微米黑&quot;;

color:rgb(0, 122, 165);
border-radius:5px;
border:2px solid rgb(0, 122, 165);
background-color: rgb(255, 255, 255);
}
QToolButton{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
font: 14pt &quot;DejaVu Sans&quot;;
color:rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}
QLabel{
font: 18pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
}
QGroupBox{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:8px;
margin-top:22px;
}
QGroupBox:title{
subcontrol-origin: margin;
left: 5px;
}</string>
  </property>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
background-color:rgb(0, 122, 165);</string>
   </property>
   <property name="text">
    <string> 系统时间</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>45</y>
     <width>1011</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEditYear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>55</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMonth">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>55</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHour">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>145</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_hour">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>145</y>
      <width>65</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 12pt &quot;DejaVu Sans&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>时</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_second">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>145</y>
      <width>65</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 12pt &quot;DejaVu Sans&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDay">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>55</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_year">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>55</y>
      <width>65</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 12pt &quot;DejaVu Sans&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>年</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSecond">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>145</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_day_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>55</y>
      <width>65</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 12pt &quot;DejaVu Sans&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>日</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMinute">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>145</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_minute">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>145</y>
      <width>65</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 12pt &quot;DejaVu Sans&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>分</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_month">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>55</y>
      <width>65</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 12pt &quot;DejaVu Sans&quot;;
color:rgb(0, 122, 165);
}</string>
    </property>
    <property name="text">
     <string>月</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="tBtnSysTimeOk">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>75</y>
      <width>92</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>设置
时间</string>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>35</y>
      <width>306</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
font: 22pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}</string>
    </property>
    <widget class="QToolButton" name="tBtn_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_9">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>150</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_0">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>215</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_6">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>80</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_3">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>10</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_del">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>215</y>
       <width>176</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtn_7">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>150</y>
       <width>80</width>
       <height>60</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbGetTime">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>476</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>2020年5月23日 13:23:25</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>365</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
